pages/account/index.vue

```diff
@@ -25,8 +25,23 @@
               <b-container>
                 <b-row class="flex-md-row">
                   <b-col cols="12">
+                    <template
+                      v-if="likedItems"
```

Comment:
Since there will always be a likedItems array (after the changes in set store), this condition will always be satisfied. As a result, the likes-spinner will never appear.

I tested it and, indeed, the empty-likes message appears while the likes are still loading, instead of the spinner